console/src/components/Services/Data/TableModify/ModifyActions.js

```diff
@@ -715,12 +715,11 @@ const setUniqueKeys = keys => ({
   keys,
 });
 
-const changeTableName = (oldName, newName, isTable) => {
+const changeTableName = (oldName, newName, isTable, objectType) => {
```

Comment:
Name `objectType` might be a bit confusing. What about `tableType`?

Reply:
Done!